sources/lib/datalab/gcp/datalab/_commands.py

```diff
@@ -40,7 +49,15 @@ def format_usage(self):
 
   @staticmethod
   def create_args(line, namespace):
-    """ Expand any meta-variable references in the argument list. """
+    """ Tokenize a magic command line expanding any meta-variable references.
```

Comment:
No - but maybe a comma.

Tokenize a magic command line, expanding ...

Because it is tokenizing and expanding